common/men_at_arms_types/wc_cultural_maa_types.txt

```diff
@@ -701,14 +701,14 @@ frost_wyrm = {
 ghoul = {
 	type = skirmishers
 	
-	damage = 17
-	toughness = 11
-	pursuit = 12
-	screen = 5
+	damage = 8
+	toughness = 5
+	pursuit = 7
+	screen = 0
 
 	terrain_bonus = {
-		taiga = { damage = 10 toughness = 5 pursuit = 10}
-		forest = { damage = 10 toughness = 5 pursuit = 10}
+		taiga = { damage = 3 toughness = 2 pursuit = 3 }
+		forest = { damage = 3 toughness = 2 pursuit = 3 }
```

Comment:
Very minor, but if we increase their damage OR pursuit by 1 for taiga and forest, they will have a balance factor in between 120%-121% which is what we are aiming for (now is about 119%)

Reply:
That's such a small difference, we have bearpaws and savages slightly below 120% as well. And with the intention of having ghouls weaker, I'd say 119.6 is already close enough to 120.

Reply:
Fair